src/app/_hooks/useToggle.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+
+const useToggle = () => {
+  const [isShow, setIsShow] = useState(false);
+
+  const handleClickOpen = () => {
+    setIsShow(true);
+  };
+
+  const handleClickClose = () => {
+    setIsShow(false);
+  };
+
+  const handleClickToggle = () => {
+    setIsShow(!isShow);
+  };
```

Comment:
[ P4 ]

useToggle에 맞게 state값이 `isShow` 보다는 `isToggle`이 좋을것 같습니다! 

또한 현재 네이밍으로는 `Modal` 혹은 `Click` 이벤트에 한정된 hook처럼 보일 수 있을것같아요!

```suggestion
  const handleToggleOn = () => {
    setIsShow(true);
  };

  const handleToggleOff = () => {
    setIsShow(false);
  };

  const handleToggle = () => {
    setIsShow(!isShow);
  };
```

으로 Toggle이라는 네이밍은어떨까요?

Reply:
오오.. 좋은 것 같습니다 !! 더 직관적이네요 ㅎㅎ 👍👍
감사합니다 !!